spec/requests/api/multiple_invites_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:"Api-Key") { Fabricate(:api_key).key }
   let(:"Api-Username") { "system" }
 
-  path "/invites.json?" do
+  path "/invites/create-multiple.json?" do
```

Comment:
Can we remove the question mark? Maybe that was added before so their wasn't a naming collision, but we shouldn't need it now.